hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsBlobClient.java

```diff
@@ -404,14 +408,9 @@ public AbfsRestOperation createPath(final String path,
       boolean isCreateCalledFromMarkers) throws AzureBlobFileSystemException {
     final List<AbfsHttpHeader> requestHeaders = createDefaultHeaders();
     if (!isNamespaceEnabled && !isCreateCalledFromMarkers) {
-      Path parentPath = new Path(path).getParent();
-      if (parentPath != null && !parentPath.isRoot()) {
-        createMarkers(parentPath, overwrite, permissions, isAppendBlob, eTag,
-            contextEncryptionAdapter, tracingContext, isNamespaceEnabled);
-      }
       AbfsHttpOperation op1Result = null;
       try {
-        op1Result = getPathStatus(path, tracingContext,
+        op1Result = getPathStatus(path, new TracingContext(tracingContext),
```

Comment:
Why do we need is implicit check as true here??
If we need to check for implicit then the Log message below needs to be changed.

Reply:
We should not create a file where a directory is already present be it implicit or explicit, hence we need implicit check as well. Updated the log